src/content/guides/environment-variables.md

```diff
@@ -3,35 +3,48 @@ title: Environment Variables
 sort: 16
 contributors:
   - simon04
+  - grisanu
 related:
-  - title: The Fine Art of the webpack 2 Config
-    url: https://blog.flennik.com/the-fine-art-of-the-webpack-2-config-dc4d19d7f172#.297u8iuz1
+  - title: The Fine Art of the webpack 3 Config
+    url: https://blog.flennik.com/the-fine-art-of-the-webpack-2-config-dc4d19d7f172#d60a
 ---
 
 To disambiguate in your `webpack.config.js` between [development](/guides/development) and [production builds](/guides/production), you may use environment variables.
 
-The standard approach in Node.js modules can be applied: Set an environment variable when running webpack and refer to the variables using Node's [`process.env`](https://nodejs.org/api/process.html#process_process_env). The variable `NODE_ENV` is commonly used as de-facto standard (see [here](https://dzone.com/articles/what-you-should-know-about-node-env)).
+The webpack command line [environment option](/api/cli/#environment-options), `--env` allows you to pass in as many environment variables as you would like. Environment variables will be made accesible in your `webpack.config.js`. For example, `--env.production` or `--env.NODE_ENV=local` (`NODE_ENV` is conventionally used to define the environment type, see [here](https://dzone.com/articles/what-you-should-know-about-node-env).)
 
-**webpack.config.js**
-
-```diff
-module.exports = {
-  plugins: [
-    new webpack.optimize.UglifyJsPlugin({
-+      compress: process.env.NODE_ENV === 'production'
-    })
-  ]
-};
+```bash
+webpack --env.NODE_ENV=local --env.production --progress
 ```
 
-Use the [`cross-env`](https://www.npmjs.com/package/cross-env) package to cross-platform-set environment variables:
+T> Setting up your `env` variable without assignment, `--env.prduction` sets `env.prduction` to `true` by default. There are also other various syntaxes that you can use. See the [webpack CLI](/api/cli/#environment-options) documentation for more information.
```

Comment:
@grisanu I would probably say 'the assignment' unless I'm missing something, and also do you mean `--env.production`?

You don't really need the 'various' really.